CONTRIBUTORS.txt

```diff
@@ -83,8 +85,11 @@ needs to be listed here.
 - Norman Maurer <norman_maurer@apple.com>
 - Ole Begemann <ole@oleb.net>
 - OleG <emptystamp@gmail.com>
+- Petro Rovenskyy <petro.rovenskyy@gmail.com>
 - Portia <portia-s@users.noreply.github.com>
 - Pushkar N Kulkarni <pushkar.nk@in.ibm.com>
+- Pushparaj Zala <przala@users.noreply.github.com>
```

Comment:
@przala if you'd prefer a real email address, please let me know which one.

Reply:
yes pls
pr92zala@gmail.com

Thank you

Reply:
done